### PR TITLE
Migrate tests/app/submitter and tests/app/utilities to pytest

### PR DESCRIPTION
**Pull request by jamesnunn**
### What is the context of this PR?
Migrate tests/app/submitter and tests/app/utilities to pytest

### How to review 
Check the tests make sense

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by jamesnunn** [comment]
[Migrate tests/app/submitter and tests/app/utilities to pytest (2 SP)](https://trello.com/c/cVNy1sNX/5168-migrate-tests-app-submitter-and-tests-app-utilities-to-pytest-2-sp)

File: tests/app/submitter/test_submitter.py [hunk 1]
```diff
@@ -1,276 +1,251 @@
...
-        blob_contents = blob.upload_from_string.call_args[0][0]
-
-        assert (
-            blob_contents
-            == b'{"feedback-type": "Feedback type", "feedback-text": "Feedback text", '
-            b'"feedback_count": 1, "feedback_submission_date": "2021-03-23", '
-            b'"form_type": "H", "language_code": "cy", "region_code": "GB-ENG", "tx_id": "12345"}'
-        )
-        assert feedback_upload is True
+def test_when_fail_to_connect_to_queue_then_published_false(
```
Comment (MebinAbraham) [inline tests/app/submitter/test_submitter.py:10]: I think classes would still help. Grouping them by submitter makes sense. At the moment, the method names aren't that clear given it isn't grouped by the submitter. That, or maybe separate files?.

Comment (MebinAbraham) [reply tests/app/submitter/test_submitter.py]: Well it's not for convenience, it's for readability and clarity. I think prefixing with the submitter solves it.

File: tests/app/submitter/test_submitter.py [hunk 2]
```diff
@@ -1,276 +1,257 @@
...
-        blob_contents = blob.upload_from_string.call_args[0][0]
-
-        assert (
-            blob_contents
-            == b'{"feedback-type": "Feedback type", "feedback-text": "Feedback text", '
-            b'"feedback_count": 1, "feedback_submission_date": "2021-03-23", '
-            b'"form_type": "H", "language_code": "cy", "region_code": "GB-ENG", "tx_id": "12345"}'
-        )
-        assert feedback_upload is True
+def test_rabbitmq_submitter_when_fail_to_connect_to_queue_then_published_false(
```
Comment (MebinAbraham) [inline tests/app/submitter/test_submitter.py:10]: The function names should be redone, they don't read well. This one can be like:
`test_rabbitmq_submitter_not_published_when_fails_to_connect_to_queue`

The same applies to most of the others, they could do with updating.

**Review by MebinAbraham — APPROVED** [review]

File: tests/app/utilities/test_strings.py [hunk 3]
```diff
@@ -1,33 +1,27 @@
...
+def test_to_bytes(input_str, bytes_str):
+    assert strings.to_bytes(input_str) == bytes_str
+
+
+@pytest.mark.parametrize(
+    "input_str,bytes_str",
+    (
+        ("hij", "hij"),
+        (b"klm", "klm"),
+        (None, None),
```
Comment (parimooa) [inline tests/app/utilities/test_strings.py:23]: could do with one more scenario `("","")`

**Review by parimooa — APPROVED** [review]
LGTM , just 1 minor comment

**Pull request merged**